coretex/cli/modules/node.py

```diff
@@ -378,13 +389,21 @@ def configureNode(config: Dict[str, Any], verbose: bool) -> None:
 
             if nearWalletId != config_defaults.DEFAULT_NEAR_WALLET_ID:
                 config["nearWalletId"] = nearWalletId
+                endpointInvocationPrice = clickPrompt(
+                    "Enter the price of a single endpoint invocation",
+                    config_defaults.DEFAULT_ENDPOINT_INVOCATION_PRICE,
```

Comment:
What happens if the price which is entered is 0? Should that be allowed? Can you check this with backend team